modules/hcp-ecs-client/services.tf

```diff
@@ -20,7 +20,7 @@ module "acl-controller" {
 }
 
 resource "aws_iam_role" "frontend-task-role" {
-  name = "frontend_${local.scope}_task_role"
+  name = "${local.secret_prefix}_frontend_${local.scope}_task_role"
```

Comment:
isn't `local.scope` the same like `local.secret_prefix`? Not literally the same, but also something random?

Reply:
Ah yeah, that's right, my bad

Looking at this again, I was actually hitting the duplicate name down in `mesh-task/iam/`:

```
// Create the execution role and attach policies
resource "aws_iam_role" "execution" {
  count = local.create_execution_role ? 1 : 0
  name  = "${var.family}-execution"
```

Where family name was not unique so the role wasn't